src/main/kotlin/no/nav/syfo/aktivitetskrav/database/AktivitetskravVarselRepository.kt

```diff
@@ -81,39 +82,49 @@ class AktivitetskravVarselRepository(private val database: DatabaseInterface) {
     ): Int =
         withContext(Dispatchers.IO) {
             database.connection.use { connection ->
-                val rowsAffected = connection.prepareStatement(Queries.setExpiredVarselPublishedAt).use {
-                    it.setObject(1, nowUTC())
-                    it.setObject(2, nowUTC())
-                    it.setString(3, publishedExpiredVarsel.varselUuid.toString())
-                    it.executeUpdate()
-                }
+                val rowsAffected =
+                    connection.prepareStatement(SET_EXPIRED_VARSEL_PUBLISHED_AT).use {
+                        it.setObject(1, nowUTC())
+                        it.setObject(2, nowUTC())
+                        it.setString(3, publishedExpiredVarsel.varselUuid.toString())
+                        it.executeUpdate()
+                    }
                 if (rowsAffected != 1) {
                     throw SQLException("Expected one row to be updated, got update count $rowsAffected")
                 }
                 connection.commit()
                 rowsAffected
             }
         }
-}
 
-private object Queries {
-    const val selectExpiredVarsler =
-        """
-            SELECT a.personident, a.uuid as aktivitetskrav_uuid, varsel.*
+    companion object {
+        private const val AKTIVITETSKRAV_WITH_HANDLED_VARSEL =
+            """
+            SELECT a.id
```

Comment:
Usikker på om det gir mening å trekke denne ut i egen variabel når `a` her kun er definert der den brukes i `SELECT_EXPIRED_VARSLER`

Reply:
Ja, her refererer ikke "a" til noen ting, så dette kan ikke virke. Bør stå "SELECT vudering.aktivitetskrav_id FROM .."

Reply:
Tenkte at det hjalp for leseligheten til queryen. Så slipper man å forholde seg til innholdet i `AKTIVITETSKRAV_WITH_HANDLED_VARSEL` fordi man vet at det er handled varsler 😊
Kan godt bare innlemme i queryen også. Hvis man kikker på kode på query nivå så er kanskje sjansen stor for at man er interessert i detaljene i en query?

Reply:
Synes det blir mer forvirrende å skille den ut siden den bare fungerer i kontekst av `SELECT_EXPIRED_VARSLER` (det er der man definerer `a`). Så da tenker jeg også den bør innlemmes der. 
Hadde den fungert som en "frittstående" query man kunne brukt f.eks. fra andre repository-funksjoner så hadde det gitt mer mening å skille den ut i en variabel :slightly_smiling_face:

Reply:
Litt enig, jeg slet først med å skjønne den queryen når jeg reviewet, ettersom `a` og `varsel` ikke var definert i den queryen. Så ja, queryen blir lang hvis man slår det sammen, men føles kanskje mer riktig å ha alt i ett 🤔